lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -320,22 +320,16 @@ function dosomething_reportback_form_submit($form, &$form_state) {
   // Save values into reportback.
   $rid = dosomething_reportback_save($values);
 
-  // Redirect to confirmation.
-  $display_permalink = variable_get('dosomething_reportback_display_permalink');
```

Comment:
I'm going to have another pr to change that to `display_share_form` today
#4162